freshservice/README.md

```diff
@@ -0,0 +1,118 @@
+
+URL
+Freshservice has 2 different versions of their API. This guide uses V2, but it is possible to use V1 with slight modifications to your JSON payload.
+
+In the URL field enter the following endpoint:
+
+https://<YOUR_DOMAIN>.freshservice.com/api/v2/tickets
+
+Custom Payload
+First, check the Use custom payload checkbox to use a custom JSON payload. If you don’t do this, the standard Datadog payload is sent to Freshservice resulting in a failed request.
+
+Next, enter a new ticket JSON payload. The following example uses only the required fields, so review Freshservice’s ticket endpoint for more options on customizing your payload:
+
+{
+  "email": "[email address to associate with ticket]",
+  "subject": "$EVENT_TITLE",
+  "description": "<img src=\"$SNAPSHOT\" /><hr/>$TEXT_ONLY_MSG",
+  "status": 2,
+  "priority": 2
+}
+Note:
+
+Values such as $EVENT_TITLE are variables used by our webhook integration. For a full list of these variables and their meaning, refer to the webhook integration tile or our webhook integration documentation.
+Manually enter an email address for the email field instead of using the variable of $EMAIL, which is only populated when mentioning the webhook in an Event Stream comment and not used within Monitor Alerts.
+The description field of the payload accepts HTML. Our $EVENT_MSG variable renders your monitor’s message in Markdown, which is not supported by Freshservice’s API, so $TEXT_ONLY_MSG is used instead, along with a graph snapshot.
+The status and priority fields are numbers mapped to different values. To see these values, review Freshservice’s ticket endpoint.
+Authentication
+Freshservice’s API uses Basic Access Authentication. Your Base64 encoded credentials should be sent in the Authorization request header. Accepted credentials are your username and password in username:password format, or your Freshservice API key.
+
+To set this up in your webhook, add the following to your Headers section:
+
+{"Authorization": "Basic <BASE64_ENCODED_CREDENTIALS>"}
+Finishing Up
+In the Webhook integration tile, click Install Integration or Update Configuration (if you previously entered a webhook definition) to save your changes.
+
+### Configuration
+
+You can now add the @webhook-<NAME> to your monitor message. The webhook is triggered when the monitor changes state.
+
+We recommend adding your at-mention inside of {{#is_alert}} or {{#is_warning}} conditionals, for example:
+
+{{#is_alert}}
+    {{host.name}} is down!
+    @webhook-freshservice
+{{/is_alert}}
+When your monitor triggers an alert, a new ticket appears in your Freshservice dashboard. If you choose not to use a conditional statement, a new ticket is created when the monitor recovers because the webhook is triggered again.

```

Comment:
```suggestion
When your monitor triggers an alert, a new ticket appears in your Freshservice dashboard. If you choose not to use a conditional statement, a new ticket is created when the monitor recovers because the webhook is triggered again.

```